01week/javascripting/js/datatypes.js

```diff
@@ -0,0 +1,133 @@
+'use strict';
+
+// Main wrapper function
+$(document).ready(function() {
+
+  init();
+
+  // Take note, number on function or style represents the test number
+
+// Button click event for test 1
+  $('.btn1').click(function() {
+    callTest1();
+  });
+
+  // Button click event for test 2
+  $('.btn2').click(function() {
+    callTest2();
+  });
+
+  // Button click event for test 3
+  $('.btn3').click(function() {
+    callTest3();
+  });
+
+  // Button click events for test 4.  Test 4 has 2 buttons.
+  $('.btn4').click(function() {
+    callTest4a();
+  });
+
+  // 2nd button click event for test 4.  Test 4 has 2 buttons.
+  $('.btn4b').click(function() {
+    let notDefined;
+    $('.sentence4b').text('Unassigned variable "notDefined" returns:  ' + callTest4b(notDefined) );
+    $('.sentence4c').text('Passing 0/0 into a function returns number:  ' + callTest4b(0/0));
+    $('.sentence4d').text('0/0 directly returns:  ' + typeof 0/0 );
+
+  });
+
+  // Button click event for test 5.
+  $('.btn5').click(function() {
+    callTest5();
+  });
+
+  // Button click event for test 678.
+  $('.btn678').click(function() {
+    callTest678();
+  });
+
+});
+
+function init() {
+  $('.sentence6').text('Boring sentence');
+  $('.sentence7').text('Boring sentence');
+  $('.sentence8').text('Boring sentence');
+
+};
+
+// Function call for test 1 - Display date and time
+function callTest1() {
+  $('#currentDate').html(`Date and time:<span class='underline'> ${Date()}</span>`);
+  return;
+}
+
+// Function call for test 2 - convert number to a string
+function callTest2() {
```

Comment:
This function will return the exact value of the input. I need this function to be able to take in a number and convert to a string. To do this assignment, I would do strict JS and not worry about jQuery or interacting with HTML.